driver-sync/src/test/functional/com/mongodb/client/unified/ErrorMatcher.java

```diff
@@ -56,6 +56,14 @@ void assertErrorsMatch(final BsonDocument expectedError, final Exception e) {
                     expectedError.getBoolean("isClientError").getValue(),
                     e instanceof MongoClientException || e instanceof IllegalArgumentException);
         }
+        if (expectedError.containsKey("errorContains")) {
```

Comment:
First use of this, so had to add support

Reply:
Ack